cpp/src/experimental/louvain.cuh

```diff
@@ -405,67 +60,35 @@ class Louvain {
       handle_(handle),
       dendrogram_(std::make_unique<Dendrogram<vertex_t>>()),
       current_graph_view_(graph_view),
-      compute_partition_(handle, graph_view),
-      local_num_vertices_(graph_view.get_number_of_local_vertices()),
-      local_num_rows_(graph_view.get_number_of_local_adj_matrix_partition_rows()),
-      local_num_cols_(graph_view.get_number_of_local_adj_matrix_partition_cols()),
-      local_num_edges_(graph_view.get_number_of_edges()),
-      vertex_weights_v_(graph_view.get_number_of_local_vertices()),
-      cluster_weights_v_(graph_view.get_number_of_local_vertices()),
-      number_of_vertices_(graph_view.get_number_of_local_vertices()),
+      cluster_keys_v_(graph_view.get_number_of_local_vertices(), handle.get_stream()),
+      cluster_weights_v_(graph_view.get_number_of_local_vertices(), handle.get_stream()),
+      vertex_weights_v_(graph_view.get_number_of_local_vertices(), handle.get_stream()),
+      src_vertex_weights_cache_v_(0, handle.get_stream()),
+      src_cluster_cache_v_(0, handle.get_stream()),
+      dst_cluster_cache_v_(0, handle.get_stream()),
       stream_(handle.get_stream())
```

Comment:
What's the purpose of having both ```handle_``` and ```stream_``` as ```stream``` is just ```handle_.get_stream()```? This is like maintaining two copies of the same variable.

Reply:
The code was adapted from a previous version before we had handle.  I can replace the stream_ references with handle_.get_stream()